textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/performancetest/regexmatcher/RegexMatcherPerformanceTest.java

```diff
@@ -0,0 +1,73 @@
+package edu.uci.ics.textdb.sandbox.performancetest.regexmatcher;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.custom.CustomAnalyzer;
+import org.apache.lucene.analysis.ngram.NGramTokenizerFactory;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+import edu.uci.ics.textdb.common.exception.StorageException;
+import edu.uci.ics.textdb.dataflow.common.RegexPredicate;
+import edu.uci.ics.textdb.dataflow.regexmatch.RegexMatcher;
+import edu.uci.ics.textdb.sandbox.medlineIndexBuilder.MedlineData;
+import edu.uci.ics.textdb.sandbox.medlineIndexBuilder.MedlineIndexWriter;
+import edu.uci.ics.textdb.storage.DataStore;
+
+/*
+ * This is a sample performance test 
+ * using Medline data and helper functions.
+ * 
+ * @author Zuozhi Wang
+ */
+public class RegexMatcherPerformanceTest {
+	
+	public static void main(String[] args) throws StorageException, IOException, DataFlowException {
+		peoformanceTest1();
+	}
+
+	public static void peoformanceTest1() throws StorageException, IOException, DataFlowException {
+		
+		Analyzer luceneAnalyzer = CustomAnalyzer.builder()
+				.withTokenizer(NGramTokenizerFactory.class, new String[]{"minGramSize", "3", "maxGramSize", "3"})
+				.build();
+		
+		long startIndexTime = System.currentTimeMillis();
+		
+		DataStore dataStore = MedlineIndexWriter.writeMedlineToIndex(
+				"/Users/georgewang/Documents/Project_Medextract/dataset/abstract_100K.txt", "./index", luceneAnalyzer);
```

Comment:
Understood.  Still it will be good to have a "common" folder for people to put their data files there, and we can use relative paths with respect to that folder.  How about under `edu/uci/ics/textdb/sandbox/data-files`?